uk.co.compucorp.civicrm.hrleaveandabsences/views/manager-leave/components/manager-leave-requests.html

```diff
@@ -118,12 +118,22 @@
                     <i class="fa fa-ellipsis-v"></i>
                   </a>
                   <ul class="dropdown-menu dropdown-menu-right" uib-dropdown-menu>
-                    <li ng-repeat="action in ctrl.actionsFor(request)">
-                      <a ng-if="action === 'cancel'" href
+                    <li ng-repeat="action in ctrl.actionsFor(request)" ng-switch on="action">
+                      <a ng-switch-when="cancel" href
                         ng-click="ctrl.action(request, action)">
                         {{action}}
                       </a>
-                      <a ng-if="action !== 'cancel'" href
+                      <a ng-switch-when="approve" href
+                        ng-click="ctrl.action(request, action)">
+                        {{action}}
+                      </a>
+                      <a ng-switch-when="reject" href
+                        ng-click="ctrl.action(request, action)">
+                        {{action}}
+                      </a>
```

Comment:
```html
<a ng-switch-when="cancel" href
  ng-click="ctrl.action(request, action)">
  {{action}}
</a>
<a ng-switch-when="approve" href
  ng-click="ctrl.action(request, action)">
  {{action}}
</a>
<a ng-switch-when="reject" href
  ng-click="ctrl.action(request, action)">
  {{action}}
</a>
```
these are the same thing, there is no point to have it repeat 3 times. It actually seems that the only one different is the `ng-switch-default` one